OracleDatabase/OracleRealApplicationClusters/README.md

```diff
@@ -0,0 +1,474 @@
+FOLLOWING FREELY AVAILABLE PATCH IS REQUIRED FOR THIS IMAGE TO WORK
```

Comment:
This should not be at the top of the README. You can reference this later when you list the binary archives that need to be downloaded. There's no need to mention "freely available" here either.